src/test/java/com/twilio/rest/api/v2010/account/usage/record/DailyTest.java

```diff
@@ -36,6 +37,25 @@ public void setUp() throws Exception {
         Twilio.init("AC123", "AUTH TOKEN");
     }
 
+    @Test
+    public void testThrowingAPIConnectionExceptionInCaseOfEmptyResponse() throws Exception {
```

Comment:
This test would be better located in [TwilioRestClientTest](https://github.com/twilio/twilio-java/blob/main/src/test/java/com/twilio/http/TwilioRestClientTest.java#L13), as these tests are autogenerated from our backend api definitions.